modules/backend/widgets/form/partials/_field_dropdown.htm

```diff
@@ -6,6 +6,15 @@
 <!-- Dropdown -->
 <?php if ($this->previewMode): ?>
     <div class="form-control"><?= (isset($fieldOptions[$field->value])) ? e(trans($fieldOptions[$field->value])) : '' ?></div>
+
+<!-- ReadOnly alternative -->
+<?php elseif ($field->readOnly): ?>
+    <div class="form-control" disabled="disabled"><?= (isset($fieldOptions[$field->value])) ? e(trans($fieldOptions[$field->value])) : '' ?></div>
```

Comment:
This would be DRYer if you used `if ($this->previewMode || $field->readOnly)` in the first conditional and then simply put a `<?= $field->readOnly ? 'disabled="disabled"' : ''; ?>` in the `<div class="form-control"` of the preview mode conditional.

Reply:
Then also put the 
```
<?php if ($field->readOnly): ?>
    <input
         type="hidden"
         name="<?= $field->getName() ?>"
         value="<?= $field->value ?>">
<?php endif; ?>
```

Reply:
Sounds good, I'll push those changes 👍

Reply:
The code would look like this with the modifications:

![image](https://user-images.githubusercontent.com/2809147/31506450-d7b6820e-af4d-11e7-849e-deec3b4f9ab4.png)